components/composer/src/init.spec.js

```diff
@@ -919,9 +919,11 @@ describe("Composer subject", () => {
   });
 
   it("Sets subject", () => {
-    cy.get("@testComponent").then((element) => {
-      element[0].value = { subject: "Test subject" };
-    });
+    cy.get("@testComponent")
+      .then((element) => {
+        element[0].value = { subject: "Test subject" };
+      })
+      .wait(500);
```

Comment:
Arbitrary waits usually lead to flaky tests. Is this waiting for a request? If it's a reactive change then a more reliable solution would be to improve the code logic so it's more predictable.